Clarify documentation about customMapStyles being only effective with Google provider (#4950)
react-native-maps/react-native-maps: README.md

}
```

> ### Customizing the map style (Google Maps provider only)

Reviewer: ```suggestion
### Customizing the map style (Google Maps only)
```

}
```

> This option only works with the Google provier. So for iOS, in addition to providing the `mapStyle` you will need to do specify the provier with the following:

Reviewer: ```suggestion
For iOS, in addition to providing the `mapStyle` you will need to set the provider to Google Provider as in:
```

> This option only works with the Google provier. So for iOS, in addition to providing the `mapStyle` you will need to do specify the provier with the following:

Reviewer 2: typo: should be "Google provider"
